security/pkg/k8s/tokenreview/client_test.go

```diff
@@ -0,0 +1,162 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tokenreview
+
+import (
+	"encoding/json"
+	"encoding/pem"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	k8sauth "k8s.io/api/authentication/v1"
+)
+
+func TestReview(t *testing.T) {
```

Comment:
client_test.go is not needed: https://github.com/istio/istio/pull/10672 has tests for the utility functions. This PR should call the ValidateK8sJwt() function of https://github.com/istio/istio/pull/10672.